vnext/local-cli/generator-windows/templates/cpp/src/App.cpp

```diff
@@ -38,3 +42,89 @@ App::App() noexcept
 
     InitializeComponent();
 }
+
+/// <summary>
+/// Invoked when the application is launched normally by the end user.  Other entry points
+/// will be used such as when the application is launched to open a specific file.
+/// </summary>
+/// <param name="e">Details about the launch request and process.</param>
+void App::OnLaunched(LaunchActivatedEventArgs const& e)
+{
+    __super::OnLaunched(e);
```

Comment:
>__super [](start = 4, length = 7)

This is not standard C++. Please do not use. 
Typically we just define an alias in header file:
using Super = ...
and the use it in code.